src/autonlp/autonlp.py

```diff
@@ -101,17 +101,31 @@ def create_project(self, name: str, task: str, language: str, max_models: int):
         self._project.refresh()
         return self._project
 
-    def create_evaluation(self, task: str, dataset: str, model: str, col_mapping: str, split: str, config: str = None):
+    def create_evaluation(
+        self, task: str, dataset: str, model: str, split: str, col_mapping: str = None, config: str = None
+    ):
         self._login_from_conf()
+
+        if task in DATASETS_TASKS:
+            task = format_datasets_task(task, dataset, config)
+            if col_mapping:
+                logger.warning("A task template from `datasets` has been selected. Deleting `col_mapping` ...")
+                col_mapping = None
```

Comment:
the column mapping is handled by `datasets.prepare_for_task` in the backend so `col_mapping` should be `None` here